rest-api-spec/src/main/resources/rest-api-spec/api/_global.json

```diff
@@ -0,0 +1,29 @@
+{
+  "description": "Parameters that are accepted by all API endpoints.",
+  "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html",
+  "params": {
+    "pretty": {
+      "type": "boolean",
+      "description": "Pretty format the returned JSON response.",
+      "default": false
+    },
+    "human": {
+      "type": "boolean",
+      "description": "Return human readable values for statistics.",
+      "default": true
+    },
+    "error_trace": {
+      "type": "boolean",
+      "description": "Include the stack trace of returned errors.",
+      "default": false
+    },
+    "source": {
+      "type": "string",
+      "description": "The URL-encoded request definition. Useful for libraries that do not accept a request body for non-POST requests."
+    },
+    "filter_path": {
```

Comment:
"rest spec" ambiguous here, meant the rest spec runner documentation: https://github.com/gmarz/elasticsearch/blob/12bbc95d93ce9cad57bcd2d1112d79f3b12dc0d9/rest-api-spec/src/main/resources/rest-api-spec/test/README.asciidoc